*** /usr/local/texlive/2023/texmf-dist/bibtex/bst/base/acm.bst	Mon Dec 13 20:20:07 2010
--- psu-acm.bst	Sat Feb 12 07:40:26 2022
***************
*** 167,180 ****
  FUNCTION {emphasize}
  { duplicate$ empty$
      { pop$ "" }
!     { "{\em " swap$ * "}" * }
    if$
  }
  
  FUNCTION {emphasizeic}
  { duplicate$ empty$
      { pop$ "" }
!     { "{\em " swap$ * "\/}" * }
    if$
  }
  
--- 167,180 ----
  FUNCTION {emphasize}
  { duplicate$ empty$
      { pop$ "" }
!     { "{\sl " swap$ * "}" * }
    if$
  }
  
  FUNCTION {emphasizeic}
  { duplicate$ empty$
      { pop$ "" }
!     { "{\sl " swap$ * "\/}" * }
    if$
  }
  
***************
*** 426,444 ****
    if$
  }
  
  FUNCTION {format.journal.vol.num.date}
  { journal empty$
      { "empty journal in " cite$ * warning$
        ""
      }
!     { journal
        volume empty$
          'skip$
!         { " " * volume * }
        if$
        number empty$
!         'emphasizeic
!         { emphasize ", " * number * }
        if$
        year empty$
          { "empty year in " cite$ * warning$ }
--- 426,451 ----
    if$
  }
  
+ FUNCTION {embolden}
+ { duplicate$ empty$
+     { pop$ "" }
+     { "{\bf " swap$ * "}" * }
+   if$
+ }
+ 
  FUNCTION {format.journal.vol.num.date}
  { journal empty$
      { "empty journal in " cite$ * warning$
        ""
      }
!     { journal emphasize
        volume empty$
          'skip$
!         { " " * volume embolden * }
        if$
        number empty$
!         'skip$
!         { ", " * number * } 
        if$
        year empty$
          { "empty year in " cite$ * warning$ }
